backend/src/logic/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Model Configuration
+model: gpt-3.5-turbo
+temperature: 1
+topP: 1
+presencePenalty: 0
+frequencyPenalty: 0
+maxTokens: 200
+
+# Prompt Configuration
+systemPrompt: |
```

Comment:
non-actionable comment: there are many ways to configure the system prompt & user input (Example is `query_message` in this [notebook](https://github.com/openai/openai-cookbook/blob/main/examples/Question_answering_using_embeddings.ipynb) where the context is fed into the user input, so it will be hard to avoid modifying the code. But I think this is fine and sufficient.